Triage/manifest.json

```diff
@@ -15,6 +15,10 @@
         "type": "string",
         "description": "API endpoint (url)",
         "format": "uri"
+      },
+      "exclude_signed": {
+        "type": "boolean",
+        "description": "Do not retrieve report with signed binary"
       }
```

Comment:
Shouldn't it be a configuration for the trigger and not the account ?

Also we may want to provide a default value to not break the existing ones

Reply:
You're right thx!

Reply:
It should be moved inside `trigger_triage.json`